### ISSUE TITLE
How to use SoftMax through the C API

### ISSUE THREAD

**Issue by fbrundu** [dmlc/xgboost#7233]
I would like to use the SoftMax implementation below through the C API. Is there a way to use it using only this API (i.e., through `#include "xgboost/c_api.h`)? Thanks

https://github.com/dmlc/xgboost/blob/3e7e426b36eb307dff64a2f81f8a5a225377891d/src/common/math.h#L49-L76

**Comment by trivialfis** [comment]
Hi fbrundu, do you mean using it for training xgboost model or using it independently?

For the former, you can call set param function, for the later it's not possible.

**Comment by trivialfis** [comment]
There's no way to do that at the moment.  You can obtain the untransformed output by specifying predict margin.  But after that, you will have to do the transform yourself.

**Comment by fbrundu** [comment]
Ok, thank you.
